settingsmeta.yaml

```
@@ -9,10 +9,10 @@ skillMetadata:
           label: Calendar Location
           value: "local"
           options: Locally on the device|local;iCal Server|server
-    - name: iCal Login
+    - name: Calendar Service
       fields:
         - type: label
-          label: <b>If you selected iCal Server</b> above, fill out the form here, to connect Mycroft to your iCal account. If you wish not to enter this information in here, you can <a href='https://bit.ly/mycroft-calendar-local'>configure the skill on your device</a>
+          label: <b>If you selected iCal Server</b> above, fill out the form here, to connect Mycroft to your iCal account. If you wish not to enter this information in here, you can <a href='https://bit.ly/mycroft-calendar-local'>configure the skill on your device</a> If you have a self-hosted calendar that supports CalDav, you can enter the information for that calendar here.
```

```suggestion
          label: <b>If you selected iCal Calendar Service</b> above, fill out the form here, to connect Mycroft to your iCal account. If you wish not to enter this information in here, you can <a href='https://bit.ly/mycroft-calendar-local'>configure the skill on your device</a>
```
I don't know why you need that last line?

---

If this skill can utilize a CalDav service, then we need to inform the user where to input their CalDav credentials.